tgstation.dme

```diff
@@ -14,7 +14,7 @@
 
 // BEGIN_INCLUDE
 #include "_maps\__MAP_DEFINES.dm"
-#include "_maps\tgstation2.dm"
+#include "_maps\runtimestation.dm"

```

Comment:
Feature!

The server tools already force box or meta (depending on servers) unless map rotation happens. So why haven't we made this the default map?